### PR TITLE
chore: enable lint for benchmark

### PR DESCRIPTION
**Pull request by zirain**
@shawnh2 please help to review the result.

### CODE REVIEW

File: test/benchmark/suite/render.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func newMarkdownStyleTableWriter(writer io.Writer) *tabwriter.Writer {
 }
 
 func renderEnvSettingsTable(writer io.Writer) {
-	_, _ = fmt.Fprintln(writer, "Benchmark test settings:", "\n")
```
Comment (shawnh2) [inline test/benchmark/suite/render.go:113]: we need `"\n"`, nor the table in markdown will not be properly rendered

File: test/benchmark/suite/render.go [hunk 2]
```diff
@@ -260,13 +260,14 @@ func writeSection(writer io.Writer, title string, level int, content string) {
...
-	_, _ = fmt.Fprintln(writer, fmt.Sprintf(`
+	summary := fmt.Sprintf("```plaintext\n%s\n```", content)
+	_, _ = fmt.Fprintf(writer, `
 <details>
 <summary>%s</summary>
 
 %s
 
-</details>`, title, fmt.Sprintf("```plaintext\n%s\n```", content)))
+</details>\n`, title, summary)
```
Comment (shawnh2) [inline test/benchmark/suite/render.go:270]:
```suggestion
</details>
`, title, summary)
```

the `"\n"` will be rendered as literal \n

**Review by shawnh2 — APPROVED** [review]
LGTM! just one nit

**Review by shawnh2 — APPROVED** [review]

**Review by zhaohuabing — APPROVED** [review]

**Pull request merged**